src/components/ImportCampaign.vue

```diff
@@ -40,24 +38,11 @@ export default {
       doing: 'nothing',
       pourcent: 0,
       error: 0,
-      log: '',
-      doScan: null
+      doScan: null,
+      time: 0
     }
   },
   methods: {
-    showLog () {
-      var args = '{}'
-      if (this.$refs.logFile.value !== '') {
-        args = JSON.stringify({'logFile': this.$refs.logFile.value})
-      }
-      axios.post('http://opv_master:5001/import/log', args)
```

Comment:
API URI should be configurable somewhere  or at list use a configuration constant (ideally in a configuration file).